Dockerfile

```diff
@@ -66,69 +117,18 @@ RUN wget -q -O  -  https://sh.rustup.rs | \
   RUSTUP_HOME=/usr CARGO_HOME=/usr \
   sh -s -- -y --profile minimal --no-modify-path
 
-# Get Chrome and Firefox (borrowed from https://github.com/SeleniumHQ/docker-selenium)
-
-ARG CHROME_VERSION="latest"
-ARG FIREFOX_VERSION="latest"
-# Note: geckodriver version needs to be updated manually
-ARG GECKODRIVER_VERSION="0.34.0"
+COPY --from=selenium-manager-image /opt/firefox /opt/firefox
+COPY --from=selenium-manager-image /opt/geckodriver /opt/geckodriver
+COPY --from=selenium-manager-image /opt/chrome /opt/chrome
```

Comment:
In the previous codebase, `google-chome` was used.
I just put default name provided by selenium manager.
Is it okay?

Reply:
I guess it is okay. I believe selenium is smart enough to locate `chrome` or `google-chrome` in the PATH.

Also the CI looks okay.